src/shared-tailwind.config.ts

```diff
@@ -0,0 +1,141 @@
+import type { Config } from 'tailwindcss';
```

Comment:
the theme and plugins are now exported separately so that they can also be used as presets in the other apps